gfsphysics/GFS_layer/GFS_typedefs.meta

```diff
@@ -3991,6 +4074,12 @@
   units = flag
   dimensions = ()
   type = logical
+[cycling]
```

Comment:
I am curious about this variable, model can cold start in cycling mode (e.g. GEFS). Is this used to identify cold start or warm start?

Reply:
This was added by @joeolson42 when he was working on the MYNN PBL/SFC improvements. It has to do with cycling runs being conceptually different from cold starts or warm starts. Maybe we don't understand how the cycling capability works in FV3 and it could have been implemented differently. I am starting an issue to collect improvements to this PR from code review.